ext/posix/posix.c

```diff
@@ -476,6 +476,9 @@ PHP_FUNCTION(posix_ttyname)
 	}
 #if defined(ZTS) && defined(HAVE_TTYNAME_R) && defined(_SC_TTY_NAME_MAX)
 	buflen = sysconf(_SC_TTY_NAME_MAX);
```

Comment:
Is `_SC_TTY_NAME_MAX` also problematic on FreeBSD (you haven't listed it in description...)? If so, is that even defined?

Reply:
Yes it's defined.
https://cgit.freebsd.org/src/tree/lib/libc/gen/sysconf.c#n445

Reply:
I have updated the description.